src/main/java/org/embulk/filter/column/ColumnFilterPlugin.java

```diff
@@ -1,5 +1,6 @@
 package org.embulk.filter.column;
 
+import com.github.kysnm.jsonpathcompiler.internal.Path;
```

Comment:
I would like to rename this to `expressions`

refer to following
https://github.com/jayway/JsonPath/blob/master/README.md#getting-started

`JsonPath expressions`

Or Would you like to `structure` ?